benchmarks/swebench/README.md

```diff
@@ -77,8 +77,9 @@ Images must be pre-built and pushed to a **public** container registry before ru
    - Build agent-server images for instances in `princeton-nlp/SWE-bench_Verified` (test split)
    - Push images to `ghcr.io/openhands/eval-agent-server` with tags like:
      ```
-     ghcr.io/openhands/eval-agent-server:{SDK_SHA}-{INSTANCE_TAG}-source-minimal
+     ghcr.io/openhands/eval-agent-server:{SDK_SHA}-{INSTANCE_TAG}-source-minimal-fixed
```

Comment:
Can we get rid of the `--fixed`? I think it is OK since this image is ONLY used for swe-bench